src/ops/deploy/op.rs

```diff
@@ -158,11 +158,7 @@ async fn create_signed_contract_tx(
     let coin_witness_index = 1;
 
     let inputs = signer_wallet
-        .get_asset_inputs_for_amount(
-            AssetId::default(),
-            DEFAULT_SPENDABLE_COIN_AMOUNT,
-            coin_witness_index,
-        )
+        .get_asset_inputs_for_amount(AssetId::default(), 1_000_000, coin_witness_index)
```

Comment:
Should we be getting rid of the const here?

Reply:
I think it is removed from the `fuels_core` so we need to remove that. See https://github.com/FuelLabs/sway/pull/2896#discussion_r983856142 for more detail.

Reply:
Hm, should we maybe put it behind a local const then for readability? eg const SPENDABLE_COIN_AMOUNT = 1_000_000

Reply:
After reading through other PRs I see in some context local consts are used and other times not, up to you @bingcicle !